pkg/doc/signature/verifier/verifier.go

```diff
@@ -10,15 +10,33 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/jsonld"
+
 	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/proof"
 )
 
+// nolint:gochecknoglobals
+var (
+	// CredentialStatusList2017 from credential status type is treated as invalid RDF
+	//  and excluded in signature verification, but it needs to be included while verifying signature.
+	// TODO to be removed as part of [Issue#1697]
+	allowedInvalidRDF = []string{
+		"<CredentialStatusList2017>",
+		"<JsonWebSignature2020>",
+		"<Ed25519VerificationKey2018>",
+		"<EcdsaSecp256k1Signature2019>",
```

Comment:
defined in https://www.w3.org/ns/did/v1

Reply:
This one will be fixed by @kdimak as part of https://github.com/hyperledger/aries-framework-go/issues/1696